node/cn/metrics.go

```diff
@@ -21,51 +21,56 @@
 package cn
 
 import (
+	"github.com/klaytn/klaytn/consensus/istanbul/backend"
```

Comment:
Just my opinion.
In long term, we need a better name of istanbul/backend like below or we can merge istanbul and istanbul/backend.

![image](https://user-images.githubusercontent.com/29390878/62502814-4eb3f880-b82c-11e9-9997-b9182af7e375.png)

Reply:
@KimKyungup I agree that `backend` is unclear name. Other than this, there are some cases similar to this one. I think it would be better to define a best practice of package naming and apply it considering making a guideline for open source contributors.